src/main/java/org/betonquest/betonquest/BetonQuest.java

```diff
@@ -528,9 +530,10 @@ public static void event(@Nullable final Profile profile, final EventID eventID)
                     "Firing event " + eventID + " for " + profile);
         }
         try {
-            event.fire(profile);
+            return event.fire(profile);
         } catch (final QuestRuntimeException e) {
             getInstance().log.warn(eventID.getPackage(), "Error while firing '" + eventID + "' event: " + e.getMessage(), e);
+            return false;
```

Comment:
This need to be true, as here it should say that BQs logic says the event is executed. When the event throws an exception, it was still executed. And the docs also need to be adjusted.

Reply:
Where in the docs do I need to adjust? The API section?

Reply:
I mean the javadocs, so it is clear for everyone who read the javadocs

Reply:
You added it to the end user documentation (.md) please remove it there and add it to the javadocs